core/digital-twin-lib/src/DAppTwin.ts

```diff
@@ -167,6 +182,9 @@ class DAppTwin extends DigitalTwin {
       (async (): Promise<void> => {
         this.favorite = await this.isFavorite();
       })(),
+      (async (): Promise<void> => {
```

Comment:
maybe we can move the initialize function bit more on top of the file. For me this is almost the same as a constructor and should be one of the first methods so...

Reply:
I always try to sort the functions by alphabet, so the initialize function will be in the middle of a file.